build/uninstall-privileged-service.ps1

```diff
@@ -0,0 +1,16 @@
+# .SYNOPSIS
+# PowerShell script to uninstall Rancher Desktop Privileged Service
+
+param($ServiceExecutable)
+
+# .SYNOPSIS
+# Uninstalls Rancher Desktop Privileged Service
+function Uninstall-PrivilegedService {
+  Param([String] $ServiceExecutable)
+  Write-Output "Uninstalling Rancher Desktop Privileged Service"
+  $UninstallCommand = "`"${ServiceExecutable}`" uninstall"
+  $Process = (Start-Process -FilePath "${ServiceExecutable}" -Verb RunAs -Wait -PassThru -ArgumentList $UninstallCommand)
```

Comment:
You're passing the executable name as an argument to itself: `privileged-service.exe privleged-service.exe uninstall`